content/telegraf/v1.21/data_formats/input/json.md

```diff
@@ -3,12 +3,15 @@ title: JSON input data format
 description: Use the JSON input data format to parse [JSON][json] objects, or an array of objects, into Telegraf metric fields.
 menu:
   telegraf_1_21:
-  
+
     name: JSON input
     weight: 70
     parent: Input data formats
 ---
 
+{{% note %}}
+The following information applies to the legacy JSON input data format. For most cases, we recommend using the [JSON v2 input data format](/telegraf/v1.20/data_formats/input/json_v2/) instead.
+{{% /note %}}
 
 The JSON input data format parses a [JSON][json] object or an array of objects
```

Comment:
Should the "JSON" link go to an anchor on the page or to another URL?

Reply:
Another URL -- this is for the legacy/v1 version, and v2 is a different file.